hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -511,32 +511,6 @@ public OptionalLong getMaxMemStoreTS() {
     return StoreUtils.getMaxMemStoreTSInList(this.getStorefiles());
   }
 
-  /**
-   * @param tabledir {@link Path} to where the table is being stored
-   * @param hri {@link RegionInfo} for the region.
-   * @param family {@link ColumnFamilyDescriptor} describing the column family
-   * @return Path to family/Store home directory.
-   * @deprecated Since 05/05/2013, HBase-7808, hbase-1.0.0
-   */
-  @Deprecated
-  public static Path getStoreHomedir(final Path tabledir,
-      final RegionInfo hri, final byte[] family) {
-    return getStoreHomedir(tabledir, hri.getEncodedName(), family);
-  }
-
-  /**
-   * @param tabledir {@link Path} to where the table is being stored
-   * @param encodedName Encoded region name.
-   * @param family {@link ColumnFamilyDescriptor} describing the column family
-   * @return Path to family/Store home directory.
-   * @deprecated Since 05/05/2013, HBase-7808, hbase-1.0.0
-   */
-  @Deprecated
-  public static Path getStoreHomedir(final Path tabledir,
```

Comment:
I think we should have a replacement of this method, instead of replacing the caller with the code of this method directly just like an inline?

Reply:
Good point. I went through the original ticket again where the deprecation happened. Seems like `HRegionFileSystem` is the intended place. Let me try that one.

Reply:
Moved it into `HRegionFileSystem` instead.